site/content/en/contributions/RELEASING.md

```diff
@@ -232,7 +241,7 @@ It's important that the world knows about the release. Use the following steps t
    Envoy Gateway v${MAJOR_VERSION}.${MINOR_VERSION} has been released: https://github.com/envoyproxy/gateway/releases/tag/v${MAJOR_VERSION}.${MINOR_VERSION}.0
    ```
 
-2. Send a message to the Envoy Gateway Slack channel. For example:
+2. Send a message to the Envoy Gateway Slack channel and the [Google Group](https://groups.google.com/g/envoy-gateway-announce). For example:
```

Comment:
it would be great if you can elaborate how to make announcement in google groups